### DISCUSSION TITLE
Google Ads implementation

### DISCUSSION THREAD

**Discussion by zurtox** [dotnet/maui#13338] [Q&A]
### Description

How is the process of making the implementation of google Ads in .Net MAUI?
We have working at implementing Google Ads in our MAUI application, unfortunately the process has been a real pain about the target frameworks of .Net 6.0 and/or 7.0.
Does it has to be implemented like Xamarin Forms or it will be a package which contains support for all the platforms in one?

### Steps to Reproduce

1. Create a .Net MAUI App
2. Add Google Ads Nuget package (but which one?)

### Link to public reproduction project repository

-

### Version with bug

7.0 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, Android

### Affected platform versions

iOS and Android

### Did you find any workaround?

Maybe implementing as Xamarin? Or stuck in Xamarin until the light gets on in any future oficial Nuget library for MAUI

**Comment by Xyncgas** [comment]
I personally don't want to see MAUI get bloated, I just want it to work as a platform so I can do whatever I want because I am a programmer. more codes to write means more IP for the company

**Comment by Kanat9494** [comment]
Hi, did you find the solution?

**Comment by pboisso** [comment]
Interested in following this topic also.

**Comment by v283** [comment]
So how is it? 
p.s I really need this  nuget
